Multisig/UI/Settings/OwnerKeyManagement/SeedPhraseViewController/SeedPhraseViewController.swift

```diff
@@ -46,9 +46,20 @@ class SeedPhraseViewController: UIViewController {
 
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
+        NotificationCenter.default.addObserver(self, selector: #selector(screenshotTaken), name: UIApplication.userDidTakeScreenshotNotification, object: nil)
         Tracker.trackEvent(trackingEvent)
         seedPhraseView.update()
     }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        NotificationCenter.default.removeObserver(self)
```

Comment:
No need for remove observer, this is done automatically

Reply:
Without manually removing it screenshot warning is shown on verification screens when taking screenshots